submissionforms.md

```diff
@@ -138,6 +138,57 @@ Provide detailed information about a specific input-form. The JSON response docu
           "languageCodes": []
         }
       ]
+    },
+    {
+      "fields": [
+        {
+          "input": {
+            "relationship": true
+          },
+          "label": "Journal",
+          "mandatory": false,
+          "repeatable": false,
+          "hints": "Select the journal related to this volume.",
+          "selectableRelationship": [
+            {
+              "relationship": "isVolumeOfJournal",
+              "filter": "creativework.publisher:somepublishername",
+              "search-configuration": "periodicalConfiguration"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "fields": [
+        {
+          "input": {
+            "type": "name",
```

Comment:
I think the type should be "relationship". But regarding relationships, we could have a special ways of pulling data, like the one being suggested of storing data on the relationship itself. But I would prefer to have:
`"type": "relationship"`

Reply:
The problem are mixed fields, that can contain either text values (in the metadata field of the item) or a relationship. The `type` attribute is for the text values.

Reply:
I based my proposal on how Entity References works on Drupal (https://www.drupal.org/docs/user_guide/en/structure-adding-reference.html), where you have a special type of field: "Entity reference", that differs from text; number;
dropdown; or image fields. I understand the need of having mixed content. And I still think a proper way to achieve it is by having new field types that enable the Relationships feature. This way it would be clear for both people and machines.